GitUI/UserControls/RevisionGrid/Graph/RevisionGraphSegment.cs

```diff
@@ -15,6 +17,7 @@
     //     *  |
     //     | /
     //     *    <- Parent
+    [DebuggerDisplay("{Child} - {Parent}")]
```

Comment:
This relation won't be obvious in the debugger, perhaps add labels and/or a direction indicator
```suggestion
    [DebuggerDisplay("child: {Child} -> parent: {Parent}")]
```

Reply:
👍 I have added the labels but have omitted the arrow because there is no obvious direction - consider "merge 'parent' into 'child'".